examples/plugins/afk.js

```diff
@@ -0,0 +1,30 @@
+function inject (bot, option) {
+  // it is a good idea to make a same space for your plugin
+  bot.afk = {
+    start,
+    stop
+  }
+
+  let afk; let rotation = 0
+
+  async function start () {
+    afk = setInterval(async () => {
+      if (rotation === 0) {
+        await bot.look(0, 0)
+        rotation = 1
+      } else {
+        await bot.look(Math.PI, 0)
+        rotation = 0
+      }
+    }, 3000)
+  }
+
+  function stop () {
+    if (!afk) return
+    clearInterval(afk)
+  }
+}
+
+module.exports = {
+  afk: inject
```

Comment:
wait so this does nothing? have you tested it?

Reply:
What do you mean?

yes, I tested it. It works 1.16.5 without any issues.

Reply:
why not call afk inject then if you are exporting it as inject?